packages/core/src/config.ts

```diff
@@ -36,8 +36,9 @@ const mainnetConfig = {
   subgraphEndpoint: "https://subgraph.satsuma-prod.com/f37507ea64fb/xai/sentry/api",
   publicRPC: "https://arb1.arbitrum.io/rpc",
   alchemyApiKey: "oD4X3JXvJclnt36mDkqnp9CO2sZkNhYT",
-  crossmintProjectId: "", //TODO Add Production Values
-  crossmintCollectionId: "", //TODO Add Production Values
+  crossmintProjectId: "9c2eb81d-3aff-4672-8b69-522c0f6e743a", //Crossmint Production Project ID
+  crossmintCollectionId: "d7e9ae28-7931-48b6-863a-d230f0743df7", //Crossmint Production Collection ID
+  crossmintClientApiKey: "ck_production_6CETYHVga5g646puou8ewwmdgw1aiCaF8rKQ6gWFHCKnhYS4aDUMML9JnTEDWR4Y1jjFCNwGkDPD5a3WdxEPebZBaXvYLmY7B8XjcTGgzz6uoCxGpeJrMrDY3ZNwDTNFqhRzqZH24uDoqSDr7tzKPaPv1Kn6idUPiH8MqF4ovdNBZiER3euXt6bxGsW1qapQFeGwWNsiQsivwBJ7nqSr87Yf", //Crossmint Production Client API Key
```

Comment:
Where are these values coming from ?
The production account will not be the same as the staging one.

Reply:
They are from the production crossmint account.